src/screens/NoConnection/index.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react'
 import { View, BackHandler } from 'react-native'
+import { SvgXml } from 'react-native-svg'
+import { BackButton } from '~/components/buttons'
 import Background from '~/components/Background'
 import { Description, PrimaryButton, Title } from '~/components'
-import styles from './styles'
+import ellipse from '~/assets/images/NoConnection/ellipse.svg'
+import phone from '~/assets/images/NoConnection/phone.svg'
+import signal from '~/assets/images/NoConnection/signal.svg'
+import styles, { ELLIPSE_HEIGHT, ELLIPSE_WIDTH, PHONE_HEIGHT, PHONE_WIDTH, SIGNAL_HEIGHT, SIGNAL_WIDTH } from './styles'
 
 const NoConnection = () => {
   return (
     <Background style={styles.container}>
-      <View>
-        <Title style={styles.text}>No hay conexion a internet. Intente nuevamente más tarde.</Title>
-        <PrimaryButton onPress={() => { BackHandler.exitApp() }}>
-          <Description white>Aceptar</Description>
-        </PrimaryButton>
+      <View style={styles.header}>
+        <BackButton style={styles.backButton} onPress={() => BackHandler.exitApp()} />
       </View>
+      <SvgXml width={ELLIPSE_WIDTH} height={ELLIPSE_HEIGHT} style={styles.ellipse} xml={ellipse} />
+      <SvgXml width={PHONE_WIDTH} height={PHONE_HEIGHT} style={styles.phone} xml={phone} />
+      <SvgXml width={SIGNAL_WIDTH} height={SIGNAL_HEIGHT} style={styles.signal} xml={signal} />
```

Comment:
You could've exported all these images as a single one from Figma. Ask me if you want to know how.